action/remove.go

```diff
@@ -18,10 +18,7 @@ var kubeGet kubeGetter = func(m string) string {
 	log.Debug("Getting manifests from %s", m)
 
 	a := []string{"get", "-f", m}
-	out, err := exec.Command("kubectl", a...).CombinedOutput()
-	if err != nil {
-		log.Err("Could not get manifest. %s", err)
-	}
+	out, _ := exec.Command("kubectl", a...).CombinedOutput()
```

Comment:
ok, fair enough. In general, I don't like seeing us ignoring errors. it would be nice to combine the two outputs on error like so:

```go
out, err := exec.Command("kubectl", a...).CombinedOutput()
if err != nil {
  return fmt.Sprintf("%s (%s)", string(out), err)
}
return string(out)
}
```